test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -611,6 +611,31 @@ class ExtProcIntegrationTest : public HttpIntegrationTest,
     }
   }
 
+  void TestGetAndFailStream() {
```

Comment:
Nit: Envoy-style, `testGetAndFailStream`, here and below.

Reply:
Done. Thanks!